go-sdk/generate-docker.sh

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env bash
-
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
-
-if ! which docker 2> /dev/null
-then
-  echo "Error: 'docker' executable not found"
-  exit 1
-fi
-
-
-UID=`id -u`
-GID=`id -g`
-
-
-read -r -d '' SCRIPT <<- END
-apt-get update &&
-apt-get -y install unzip &&
-apt-get -y install libicu-dev &&
-chmod u+rw /registry &&
-cd /registry/go-sdk &&
-make generate &&
-make format &&
-groupadd -g $GID -o localuser &&
-useradd -m -u $UID -g $GID -o -s /bin/bash localuser &&
```

Comment:
This was breaking for me in a thousand different ways ...
The new approach is that you need "one" version of Go installed, and the rest is self-contained.